portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +30,41 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Himani!");
+
+    // Create a list of hard-coded comments.
+    List<Comment> comments = new ArrayList<>();
+
+    String name1 = "Himani Yadav";
+    String message1 = "Wow, this is a great website :)";
+    Comment comment1 = new Comment(name1, message1);
+
+    String name2 = "Jon Snow";
+    String message2 = "I know nothing :)";
+    Comment comment2 = new Comment(name2, message2);
+    
+    String name3 = "Arya Stark";
+    String message3 = "Winter is coming.";
+    Comment comment3 = new Comment(name3, message3);
+
+    comments.add(comment1);
+    comments.add(comment2);
+    comments.add(comment3);
```

Comment:
Can we maybe abstract this into a helper method so that you could add n comments without having n "comments.add(commentX);" lines?